cabal.project

```diff
@@ -31,30 +31,37 @@ package bittide
 
 package clash-prelude
   flags: -multiple-hidden
+  ghc-options: +RTS -qn4 -A128M -RTS -j4
+
+package clash-lib
+  ghc-options: +RTS -qn4 -A128M -RTS -j4
+
+package clash-ghc
+  ghc-options: +RTS -qn4 -A128M -RTS -j4
 
 package bittide
-  ghc-options: +RTS -xp -RTS
+  ghc-options: +RTS -xp -RTS +RTS -qn4 -A128M -RTS -j4
```

Comment:
Did you consciously do `+RTS ... -RTS +RTS ... -RTS` or can it be shortened to `+RTS ... ... -RTS`?

(this goes equally for all following lines)

Reply:
It can be shortened!